### DISCUSSION TITLE
Application accent color

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#256] [Q&A] [answered]
Cruiser likes customizations, so users can select freely the accent color that appears everywhere in the app.

Will be available in "Settings | Application | Appearance | App color".
(and will replace the limited menu color)

<img src="https://github.com/devemux86/cruiser/assets/3484020/91615b9e-2185-4dda-92b1-acd0a5a33c9a" width="25%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/4aea0c77-6356-4018-814e-ab6aed1521e0" width="25%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/e34ffab3-5d3a-46f7-82b7-e5aecb37784e" width="25%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/60684333-ddcb-48e1-abfe-3aa86033492b" width="25%"/>
<img src="https://github.com/devemux86/cruiser/assets/3484020/fd299003-c388-45a1-a22f-3cc02d30c0e6" width="25%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/aeda5ac4-5cc8-4b08-bd39-428bd96d96ea" width="25%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/9d8d1b47-8569-469e-891e-453a39fbc6c8" width="25%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/5a6d78f3-7df1-4698-9edc-0db06af7f30c" width="25%"/>

**Comment by MotoUKRider** [comment]
This looks really nice!

Is there going to be a way to tie this in with my [earlier thread](https://github.com/devemux86/cruiser/discussions/126) about the day/night light-sensor switching?

The thinking being that if it gets dark during a journey, the last thing you want is a bright map (or menu) filling your vision and some automagical sorcery with the light-sensor would be nice.  Perhaps we could customize/pick our "Day" and "Night" versions using the above screen configs?

**Reply by devemux86** [reply]
@MotoUKRider it is already in my list.

It is a bit complicated as it involves app + map + user options for day or night, so needs work how to implement it.

Better discuss this in its thread: :slightly_smiling_face: 
- https://github.com/devemux86/cruiser/discussions/126

**Comment by Cer-o** [comment]
We must be careful not to overcomplicate the application. You can't please everyone.

**Comment by devemux86** [comment, marked as answer]
Application accent color is available in [Cruiser 3.0.27](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

"Settings | Application | Appearance | App color"

**Reply by DolomitesTom** [reply]
With the update the map colors don't work anymore, no high contrast, gray-scale or other colour settings

**Reply by devemux86** [reply]
This setting is for the application color.

The map color is always selected in "Settings | Map | Appearance".

**Reply by Ciutnik** [reply]
@DolomitesTom 
> With the update the map colors don't work anymore, no high contrast, gray-scale or other colour settings

I noticed that you have to reactivate the downloaded map (the online map turned on by itself). Map colors still only work with offline maps - nothing has changed here.

@devemux86 
Many thanks for another great update! 👍

**Reply by devemux86** [reply]
> I noticed that you have to reactivate the downloaded map (the online map turned on by itself).

This is strange, nothing has changed here. :thinking:

**Reply by DolomitesTom** [reply]
Yes this was the problem, maps were all deactivated

**Comment by DolomitesTom** [comment]
Changing the application colour again deactivates all maps

**Reply by devemux86** [reply]
I can't reproduce it and I like to change the app color very often. :slightly_smiling_face: 
Each time all offline maps remain loaded.

**Reply by devemux86** [reply]
If you are sure that this happens, please send the app .pref file and describe with details the exact steps.

**Reply by devemux86** [reply]
Note that some settings like app theme or app color restart the app for the changes to become activated.

**Reply by devemux86** [reply]
Also all POI files remain loaded here.
What happens with POI files in your devices?

**Reply by Ciutnik** [reply]
In my case, even exiting the app deactivates the map and POI 😳

**Reply by devemux86** [reply]
Please let's continue the discussion here:
(as it is not related with the app color)

- https://github.com/devemux86/cruiser/discussions/262
